CHANGELOG.md

```diff
@@ -3,6 +3,33 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2021-11-19
+
+### Major enhancements
+
+* Pipeline has been re-implemented in [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html)
+* Software containers are now obtained from [Biocontainers](https://biocontainers.pro/#/registry)
+* Update pipeline template to nf-core/tools `1.12.1`
+* [#77](https://github.com/nf-core/nanoseq/issues/77) - Skipped alignment steps
+* [#97](https://github.com/nf-core/nanoseq/issues/97) - Add optional DNA cleaning option
+
+### Parameters
+
+* Added `--run_nanolyse` to run NanoLyse for DNA cleaning of FastQ files
+* Added `--nanolyse_fasta` to provide a fasta file for nanolyse to filter against
+
+### Software dependencies
+
+| Dependency              | Old version | New version |
+|-------------------------|-------------|-------------|
+| `bioconductor-bambu`    | 1.0.0       | 1.0.2       |
+| `nanolyse`              |             | 1.2.0       |
+| `r-base`                | 4.0.3       | 4.0.2       |
```

Comment:
Just curious: why is there is change back in r-base version? Anything that didn't run in 4.0.3?

Reply:
The [r-base version was changed](https://github.com/nf-core/nanoseq/commit/b5bf1bc861602163fd4f02fcaabac045d036654c#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed) because `DEXseq` in r-4.0.3 did not work with the test data